straxen/plugins/peaklets/peaklets.py

```diff
@@ -348,9 +372,15 @@ def compute(self, records, start, end):
         counts = np.diff(counts, axis=1).flatten()
         peaklets["n_hits"] = counts
 
-        # Drop the data_top field
-        if n_top_pmts_if_digitize_top <= 0:
-            peaklets = drop_data_top_field(peaklets, self.dtype_for("peaklets"))
+        # Drop the data_top field and waveform start field if needed
+        peaklets = drop_data_top_field(
+            peaklets,
+            strax.peak_dtype(
```

Comment:
Why can not use `self.dtype_for("peaklets")`?

Reply:
Done!